### PR TITLE
added footer with links to fb page, summit site, NSP site. fixed some…

### PR DESCRIPTION
**Pull request by davidfunnell**
-Added basic footer to layout for all pages
-Added National Ski Patrol logo and link to site (This is on the Hyak site so I assume this is ok to do?)
-Added Summit at Snoqualmie logo and link to site (This is on the Hyak site so I assume this is ok to do?)
-Added Facebook F logo that links to Snoqualmie Pass West - Volunteer Ski Patrol page
-Updated general tailwind styling for footer
-Updated MainNavBar to mainNavBar

### CODE REVIEW

File: src/components/UI/FacebookLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Link from "next/link";
+
+const FacebookLogo = () => {
+  return (
+    <Link
+      href="https://www.facebook.com/spvsp"
```
Comment (psan2) [inline src/components/UI/FacebookLogo.tsx]: Let's move these `href`s to an imported `constants.ts` file at root. That'll let us reuse them for SEO purposes when we get around to adding some SEO goodness via [next/head](https://nextjs.org/docs/pages/api-reference/components/head)

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import SnoqualmiePassLogo from "./UI/SnoqualmieLogo";
+import FacebookLogo from "./UI/FacebookLogo";
+import NSPLogo from "./UI/NSPLogo";
+
+const Footer = () => {
+  return (
+    <footer className="bg-zinc-800  w-full max-w-screen-xl mx-auto">
+      <hr className=" mx-auto border-gray-700" />
```
Comment (psan2) [inline src/components/Footer.tsx]: See if you can manually format this file? Or maybe just resave to get the formatter to hit it again. There's some weird spacing bidness here that prettier should take care of (more than just nitpicking, seeing one spot where a formatter would have caught something is generally an indication that the entire file probably didn't get formatted, so just a general code smell)

**Comment by psan2** [comment]
Lgtm! Only thing I'd say is toss a screenshot into the body of the PR when you're adding visual components, saves the reviewer having to spin it up and look.

File: src/components/Footer.tsx (hunk 2 above)
Comment (davidfunnell) [reply src/components/Footer.tsx]: For some reason my prettier isn't formatting any CSS? I'll have to look into why its doing that. I've fixed it manually though!

**Comment by davidfunnell** [comment]
image

**Review by psan2 — APPROVED** [review]

**Pull request merged**